index.html

```diff
@@ -175,6 +175,178 @@ <h5 class="сustomer-кeviews">Отзывы</h5>
+                        <span class="choice-of-payment-method" ></span>
+                        Банковский перевод
+                    </label>
+                    </div>
+                    <div class="payment-method-item">
+                        <label class="label-box">
+                        <input type="radio" class="input-radio-payment" name="radio-bank-transfer">
+                        <span class="choice-of-payment-method"></span>
+                        Банковский перевод
+                    </label>
+                    </div>
+                </div>
+                <h1 class="additional-services-text">Дополнительные услуги</h1>
+                <div class="additional-services-wrapper">
+                    <div class="additional-services-item">
+                        <input class="additional-services-checkbox" type="checkbox" id="cheackbox_call">
+                        <label class="additional-services-label" for="cheackbox_call">Вызов замерщика</label>
+                    </div>
+                    <div class="additional-services-item">
+                        <input class="additional-services-checkbox" type="checkbox" id="cheackbox_packaging">
+                        <label class="additional-services-label" for="cheackbox_packaging">Упаковка для хрупких предметов</label>
+                    </div>
+                    <div class="additional-services-item">
+                        <input class="additional-services-checkbox" type="checkbox" id="cheackbox_crane">
+                        <label class="additional-services-label" for="cheackbox_crane">Кран</label>
+                    </div>
+                    <div class="additional-services-item">
+                        <input class="additional-services-checkbox" type="checkbox" id="cheackbox_furniture">
+                        <label class="additional-services-label" for="cheackbox_furniture">Сборка мебели</label>
+                    </div>
+                    <div class="additional-services-item">
+                        <input class="additional-services-checkbox" type="checkbox" id="cheackbox_subjects">
+                        <label class="additional-services-label" for="cheackbox_subjects">Крупногабаритные предметы</label>
+                    </div>
+                </div>
+                <div class="time-button-wrapper">
+                    <p class="time">05:37:25</p>
+                    <div class="button-btn-order">
+                        <button class="btn-order">Заказать со скидой</button>
+                        <div class="shadow-btn-order"></div>
+                    </div>
+                </div>
+                <button class="btn-mobile-order">Перезвонить мне</button>
+                <div class="chekbox-data-processing">
+                    <input class="checkbox" type="checkbox" id="data-processing">
+                    <label class="checkbox-cheack" for="data-processing">Даю согласие на обработку данных</label>
```

Comment:
**cheack**
use google translate, if you have problem with english, but don't use misspelled words